src/chart.zoom.js

```diff
@@ -201,8 +201,13 @@ function panIndexScale(scale, delta, panOptions) {
 
 function panTimeScale(scale, delta, panOptions) {
 	var options = scale.options;
-	options.time.min = rangeMinLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) - delta));
-	options.time.max = rangeMaxLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - delta));
+	
+	var newMax = rangeMaxLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - delta));
+	var newMin = rangeMinLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) - delta));
+	if (newMax != options.time.max && newMin != options.time.min) {
```

Comment:
Maybe I'm using a wrong approach? (checking options.time.max for undefined doesn't look that nice)
I could also check, if the calculated new max/min is different than the output of the function rangeMax/Min/Limiter(...). Only if nothing has been cut by the function, we could set options.time.max/.min.

Something like:
```
var options = scale.options;

var targetMax = scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - delta);
var newMax = rangeMaxLimiter(panOptions, targetMax);
var targetMin = scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) - delta)
var newMin = rangeMinLimiter(panOptions, targetMin);
	
// Only set min/max, if target-values are equal to the result-values of the range-limiter-functions
if (newMax == targetMax && newMin == targetMin) {
	options.time.max = newMax;
	options.time.min = newMin;
}
```

What's your opinion?